docs/REST API/rest-api.md

```diff
@@ -878,9 +878,6 @@ None.
     "additional_queries": null
   },
   "mdm": {
-    "apple_bm_default_team": "",
-    "apple_bm_terms_expired": false,
-    "enabled_and_configured": true,
```

Comment:
Plan is to deprecate the `apple_bm_default_team`, `apple_bm_terms_expired`, `apple_bm_enabled_and_configured` flags (and remove them from docs) but support them for backwards compatibility.

What does support them mean?
- `apple_bm_default_team` - TODO
- `apple_bm_terms_expired` - set to `true` is there's one or more ABM tokens w/ terms expired
- `apple_bm_enabled_and_configured`- set to `true` is there's one or more ABM tokens

Reply:
Website redirect for the error message is added here: https://github.com/fleetdm/fleet/pull/21043/files#diff-0c6120927d3e65309562b5b15b261d3298d05fcb0ff18e5b6512ee683b7ad6b3R533

Reply:
@noahtalerman regarding `apple_bm_default_team`, currently this team is used to enroll devices regardless if it is macos, ipad or iphone. You mention:

> this value populates the macos_team for the one ABM token

Does that mean we leave ipados and ios teams to "No team" when we migrate the one existing ABM token? This would be a change vs how it currently behaves (although we probably have few ipad/ios devices atm), so we want to double-check that it's the intended behavior.

Reply:
> currently this team is used to enroll devices regardless if it is macos, ipad or iphone.

> This would be a change vs how it currently behaves

@mna really good point. Even though there are few iPhones and iPads I think it makese sense to maintain the current behavior. 

So, instead of `apple_bm_default_team` only populating `macos_team`, I think let's update this to populate all 3: `macos_team`, `ios_team`, and `ipados_team`.

I updated the dev note.

Reply:
(hiding these comments to clean up the PR)